### PR TITLE
Edited `make gendependabot` Task to Enable Dependabot for GitHub Actions

### PR DESCRIPTION
**Pull request by andresbeckruiz**
**Description:** 
Enabled Dependabot for GitHub Actions by editing the `make gendependabot` task in Makefile. For now, I have set the update frequency to weekly (this can be easily changed). This PR reflects changes made in opentelemetry-collector-contrib as well. 

**Link to tracking Issue:** 
[#3445](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/3445) in contrib repo

**Testing:**
All tests running `make` passed, no new tests were added.

**Documentation:** 
N/A

### CODE REVIEW

**Comment by bogdandrutu** [comment]
Can you also run the command :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 ## 💡 Enhancements 💡
 
 - Add `doc.go` files to the consumer package and its subpackages (#3270)
+-Enable Dependabot for Github Actions (#3312)
```
Comment (bogdandrutu) [inline CHANGELOG.md:13]: Add a space between - and E

**Comment by bogdandrutu** [comment]
This needs a rebase to pickup latest changes from head that fix tests

**Comment by andresbeckruiz** [comment]
@bogdandrutu does everything look good now?

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**